.travis.yml

```diff
@@ -1,23 +1,25 @@
 language: python
-services:
-  - redis-server
-matrix:
-  fast_finish: true
-  include:
-  - python: "2.7"
-  - python: "3.4"
-  - python: "3.5"
-  - python: "3.6"
-  - python: "3.7"
-    dist: xenial
-  - python: "pypy"
+fast_finish: true
+python: 
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
+  - 3.7
+  - pypy
+env: 
+  - RQ_VERSION="rq==0.12"
+  - RQ_VERSION="rq==1.0"
+  - RQ_VERSION="rq==1.1"
+  - RQ_VERSION="git+https://github.com/rq/rq.git"
```

Comment:
Please add egg name, so It resolves faster.

Reply:
Edited.